server/server/settings.py

```diff
@@ -216,14 +220,14 @@ def resolver_context_processor(request):
 # If you are running FuzzManager behind a TLS loadbalancer,
 # uncomment the next line to let Django know that it should
 # behave as if we were using HTTPs.
-#SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+# SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
```

Comment:
I don't like this style change. Commented code shouldn't be prefixed with whitespace. Same applies to commented code in other files (not just settings). Can we disable this in the linter?